csharp/src/Glacier2/SessionHelper.cs

```diff
@@ -220,20 +220,22 @@ private void
         Debug.Assert(router != null);
         Ice.Connection conn = router.ice_getCachedConnection();
         string category = router.getCategoryForClient();
-        int acmTimeout = 0;
+        int remoteIdleTimeout = 0;
         try
         {
-            acmTimeout = router.getACMTimeout();
+            remoteIdleTimeout = router.getACMTimeout();
         }
         catch (Ice.OperationNotExistException)
         {
         }
 
-        if (acmTimeout <= 0)
+        if (remoteIdleTimeout <= 0)
         {
-            acmTimeout = (int)router.getSessionTimeout();
+            remoteIdleTimeout = (int)router.getSessionTimeout();
         }
 
```

Comment:
We don't use ` remoteIdleTimeout`. Why do we keep reading it, is it for the future check mentioned in the TODO?

Reply:
Yes it is.